installer/PowerToysSetup/Product.wxs

```diff
@@ -527,6 +420,7 @@
     <Binary Id="PTCustomActions" SourceFile="$(var.PowerToysSetupCustomActions.TargetPath)" />
   </Fragment>
 
+  <!-- Installation directory structure -->
```

Comment:
Looking at the directory structure, is it possible to define the directory hierarchy within each of the module .wxs files?

For example, in `ImageResizer.wxs` ImageResizer could have its directory structure defined by a `DirectoryRef` to `ModulesInstallFolder` instead of `Product.wxs`.

```
<DirectoryRef Id="ModulesInstallFolder">
   <Directory Id="ImageResizerInstallFolder" Name="$(var.ImageResizerProjectName)">
           <Directory Id="ImageResizerAssetsFolder" Name="Assets" />
     </Directory>
</DirectoryRef>
```

I'm not sure whether it's doable or what added complexity it brings.

Reply:
I can try that and see what it looks like. My thinking was to have a directory structure in one place to be able to grasp it fully right away

Reply:
Yeah, that makes sense now as I think about it.